### PR TITLE
fake factor application

### PR DESCRIPTION
**Pull request by minerva1993**
Including stat. unc.
Systematic (difference from closure) is not updated...to be done later

Additional plotit flag (-q to include QCD samples, -d to use DY inclusive sample) added.
Now plotit reads genevents (in the configuration node, 'generated-events-histogram: 'hcounter'' and 'generated-events-bin: 2') from the root file. The written number in file yaml will override this, it is because WJets_HT0to100 is taken from inclusive sample, needed to modify generated events as well.

### CODE REVIEW

**Comment by easilar** [comment]
Do you still want to update the PR, or should I review to merge ?

**Comment by minerva1993** [comment]
> Do you still want to update the PR, or should I review to merge ?

@easilar Hi, please leave it opened. Thanks!

File: nanoaodframe/plot_run2.py [hunk 1]
```diff
@@ -81,35 +88,31 @@
   type: signal
   pretty-name: 'LFVSTcv'
   cross-section: 0.0368
-  generated-events: 7789000.0
-  #scale: 500
+  #generated-events: 7789000.0
```
Comment (easilar) [inline nanoaodframe/plot_run2.py:91]: Sorry, I missed something, why did we commentout this ?

**Comment by minerva1993** [comment]
I think it is good to merge this PR, if everything is fine. 
The plots are produced at the latest commit of this PR

**Review by easilar — APPROVED** [review]

**Pull request merged**